packages/jaeger-ui/package.json

```diff
@@ -15,8 +15,6 @@
     "@babel/preset-env": "^7.20.2",
     "@babel/preset-react": "^7.18.6",
     "@babel/preset-typescript": "^7.21.0",
-    "@svgr/babel-plugin-transform-svg-component": "^8.0.0",
-    "@svgr/babel-preset": "^8.0.0",
```

Comment:
how do we know these are not used by babel toolchain?

Reply:
According to their NPM Readme's, they need to be included inside the `.babelrc` or in our case, `babel-transform.js` file.

Reply:
Also, according to its implementation guide, it can only be used after defining the @svgr/{plugin-name} in some file, which I couldn't find anywhere inside the codebase.

Reply:
I checked it in the git history. Here's it is https://github.com/jaegertracing/jaeger-ui/pull/1226/files#diff-7efdc547c40e0d5e99b6ecbdba5ac9ae13de8f3d2b1ca15332363868462176e2R18-R19

It was at that time also not used anywhere.